File: app/src/main/java/com/registrationmodule/room/UserRoom.java
package com.registrationmodule.room;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;
import io.reactivex.annotations.NonNull;

@Entity(tableName = "User")
public class UserRoom {

    @PrimaryKey
    @NonNull
    Long id;

    String userName;

    String password;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: app/src/main/java/com/registrationmodule/room/UserRoom.kt
package com.registrationmodule.room

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import io.reactivex.annotations.NonNull

@Entity(tableName = "User")
class UserRoom {

    @PrimaryKey
    @NonNull
    var id: Long? = null

    var userName: String = ""

    var password: String = ""
}
